patches/server/0004-Paper-config-files.patch

```diff
@@ -1200,11 +1200,11 @@ index 0000000000000000000000000000000000000000..518a00886f7bde1de95150366541fc99
 +        if (needsConverting(legacyConfig)) {
 +            try {
 +                if (Files.exists(configDir) && !Files.isDirectory(configDir)) {
-+                    throw new RuntimeException("Paper needs to create a '" + CONFIG_DIR + "' folder in the root of your server. You already have a non-directory named '" + CONFIG_DIR + "'. Please remove it and restart the server.");
++                    throw new RuntimeException("Paper needs to create a '" + configDir.getFileName() + "' folder in the root of your server. You already have a non-directory named '" + configDir.getFileName() + "'. Please remove it and restart the server.");
```

Comment:
I don't think the config folder will necessarily be in the server root anymore?

Reply:
I don't know this option works like other file settings (paper-settings etc..) and use joptsimple. I never test if input like ../dir or absolute path directly are allowed or not. We can choose to restrict the folder to the server root or just allow both ways and change the exception message depending of where is the final path ?

Reply:
Yes normally fixed in the latest changes